app/main.py

```diff
@@ -16,4 +16,19 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [],
+    "immutable": [],
+}
+sorted_variables["mutable"].extend([
+    collection_of_coins,
+    marks,
+    my_favourite_films
+])
+sorted_variables["immutable"].extend([
+    lucky_number,
+    pi,
+    one_is_a_prime_number,
+    name,
+    profile_info
+])
```

Comment:
you can write simply like this

```suggestion
sorted_variables = {
    "mutable": [my_favourite_films, marks, collection_of_coins],
    "immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info]
}
```